views/update/update-filing-date.html

```diff
@@ -20,7 +20,13 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
     <form method="post">
       <p class="govuk-body">All the information in this update statement must be correct on this date.
       Verification checks must be completed no more than 3 months before the statement date.</p>
-      {% set filingDateLabelText = "What is the date of the update statement?" %}
+
+      <div class="govuk-inset-text">
+        <p>The date you enter <strong> must be on or before the overseas entity's update statement date, </strong> or your filing may be rejected.</p>
```

Comment:
Having a space before and after the '<strong>' markup would probably result in have two spaces between the words. What does it look like locally?

Reply:
Initially I had it as just the single space, but after a quick look it seems that HTML multiple spaces are collapsed in to one single space. I have checked locally and it shows as a single space. I did this for readability but can change it to a single space?

Reply:
For consistency, I have reverted it to single space.